src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -145,4 +145,18 @@ internal partial class Interop
     /// <returns>1 if rising edge event occured, 2 on falling edge, -1 on error.</returns>
     [DllImport(LibgpiodLibrary, SetLastError = true)]
     internal static extern int ReadEventForLine(SafeLineHandle line);
+
+    /// <summary>
+    /// List all GPIO chips.
+    /// </summary>
+    /// <returns>The number of GPIO chips, -1 if an error occurred.</returns>
+    [DllImport(LibgpiodLibrary, SetLastError = true)]
+    internal static extern int GetNumberOfChips();
```

Comment:
I think this isn't used anymore so you can perhaps remove it

Reply:
Thanks for the review @krwq 
I remove it in d37106a